app/controllers/manage_cmr/order_options_controller.rb

```diff
@@ -242,10 +241,10 @@ def legacy_edit_order_option
   end
 
   def update_order_option
-    order_option_id = params[:id]
+    order_option_concept_id = params[:id]
```

Comment:
Dosen't the params here need to be` :concept-id` ?

Reply:
this parameter is part of Rails path, defined as 'id' by the framework. The value is what you put in.